pycode/memilio-epidata/memilio/epidata/transformNPIData.py

```diff
@@ -551,13 +558,14 @@ def transform_npi_data(fine_resolution=2,
         # extract variable names for main categories
         npi_desc = list(df_npis_desc["Variable"][npi_codes_sorting])
 
+    del df_npis_desc
+
     # combine NPI codes and descriptions to ensure that both are ordered
     # the same way; delete npi_codes or npi_desc for not using hereafter
     idx_codes_retained = ~pd.Series(npi_codes).isin(codes_dropped)
-    npis_dummy = {
+    npis = pd.DataFrame({
```

Comment:
why renaming this npis and deleting afterwards? Could just have been npis_dummy...